wiki/rogue/compilers.md

```diff
@@ -0,0 +1,4 @@
+The compilers available on ne.rogue are:
+* GCC
+* Clang
+* ICC?
```

Comment:
I would put a disclaimer than to use the intel compilers, you need to contact cluster support. Might be useful to put together something detailing the pros/cons of using the different compilers.